config/software/perl.rb

```diff
@@ -35,8 +35,13 @@
   env = with_standard_compiler_flags(with_embedded_path)
 
   solaris_mapfile_path = File.expand_path(Omnibus::Config.solaris_linker_mapfile, Omnibus::Config.project_root)
-  if solaris2? && File.exist?(solaris_mapfile_path)
-    cc_command = "-Dcc='gcc -static-libgcc -Wl,-M #{solaris_mapfile_path}"
+  if solaris_10?
+    cc_command = "-Dcc='gcc -static-libgcc'"
+    if File.exist?(solaris_mapfile_path)
```

Comment:
Yeah I will need it for the moment. I have been cutting it as much as possible because it's kind of a beast to compile (obviously) but still required.